feature-libs/checkout/b2b/root/facade/checkout-cost-center.facade.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Cart, CostCenter, facadeFactory, QueryState } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { CHECKOUT_B2B_CORE_FEATURE } from '../feature-name';
+
+@Injectable({
+  providedIn: 'root',
+  useFactory: () =>
+    facadeFactory({
+      facade: CheckoutCostCenterFacade,
+      feature: CHECKOUT_B2B_CORE_FEATURE,
+      methods: ['setCostCenter', 'getCostCenterState'],
+    }),
+})
+export abstract class CheckoutCostCenterFacade {
+  /**
+   * Set cost center to cart
+   * @param costCenterId : cost center id
+   */
+  abstract setCostCenter(costCenterId: string): Observable<Cart>;
```

Comment:
as mentioned earlier, shouldn't it be Observable<unknown> ?

Reply:
`unknown`? Already answered?